lib/ferver/hash_cache.rb

```diff
@@ -0,0 +1,31 @@
+module HashCache
+  require 'yaml/store'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.